simulator/package.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>simulator</name>
+  <version>0.0.0</version>
+  <description>The simulator package</description>
+
+  <maintainer email="dvyanshu@iitk.ac.in">Divyanshu Gangwar</maintainer>
+
+  <license>TODO</license>
+
+  <author email="dvyanshu@iitk.ac.in">Divyanshu Gangwar</author>
+
+
+  <buildtool_depend>catkin</buildtool_depend>
+  <build_depend>roscpp</build_depend>
```

Comment:
just put <depend>roscpp</depend> instead of all these depends, these depends will be added automatically then.